selfdrive/camerad/cameras/camera_webcam.cc

```diff
@@ -19,13 +19,17 @@
 #pragma clang diagnostic pop
 
 
-extern ExitHandler do_exit;
+// id of the video capturing device
+#define ROAD_CAMERA_ID 1
```

Comment:
should we use environment variables here then fallback to 1 and 2 if they're not set?